packages/story-editor/src/utils/storyPageToDataUrl.ts

```diff
@@ -48,7 +53,7 @@ async function storyPageToDataUrl(page, { width = 400, ...options }) {
     height: width * (1 / PAGE_RATIO),
     canvasHeight: width * (1 / PAGE_RATIO),
     canvasWidth: width,
-  });
+  } as Options);
```

Comment:
Curious, what's not matching without casting?

Reply:
Curious as well. Casting hides potential errors.

Reply:
Removed now --- I don't recall the error when I added that but was fixed along the way with some other earlier commits.